### PR TITLE
NH M1.1

### PR DESCRIPTION
**Pull request by MarcoOl94**

### CODE REVIEW

File: test/config/nginx/templates/default.conf.tmpl [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root /usr/local/openresty/nginx/html;
+    }
+}
+
+server {
+    server_name _; # This is just an invalid value which will never trigger on a real hostname.
+    listen 80;
+    listen 443 ssl;
```
Comment (ysibirski) [inline test/config/nginx/templates/default.conf.tmpl:84]: add new line 85 `http2 off;`

File: test/config/nginx/templates/default.conf.tmpl [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    if ($cloudflare_ip != 1) {
+        return 444;
+    }
+
+    return 301 https://$host$request_uri;
+}
+
+server {
+    server_name $NGINX_DOMAIN;
+    listen 443 ssl;
```
Comment (ysibirski) [inline test/config/nginx/templates/default.conf.tmpl:113]: add new line 113 `http2 off;`

**Review by ysibirski — CHANGES_REQUESTED** [review]
2 small additions for NGINX config

**Review by lander86 — APPROVED** [review]
LGTM

File: test/config/nginx/templates/default.conf.tmpl [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
```
Comment (ysibirski) [inline test/config/nginx/templates/default.conf.tmpl]: Add `http2 off;` after line 113

**Review by ysibirski — CHANGES_REQUESTED** [review]
One more change under nginx config is required

**Review by ysibirski — APPROVED** [review]
LGTM

**Pull request merged**